src/telemetry.ts

```diff
@@ -36,4 +49,80 @@ export class NullTelemetryReporter implements ITelemetryReporter {
 
 }
 
+export const DefaultTelemetryInterval = 10000;
```

Comment:
I'd rename this to DefaultTelemetryIntervalInMilliseconds to make it clear what the unit is.

Reply:
Done.